sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractRulesDefinition.java

```diff
@@ -108,18 +119,18 @@ private RuleMetadata readRuleMetadata(String ruleKey) {
     String type;
     SecurityStandards securityStandards = new SecurityStandards();
 
-    String getKey() {
-      return sqKey;
-    }
     boolean isSecurityHotspot() {
       return SECURITY_HOTSPOT.equals(type);
     }
   }
 
-  // for deserialization purposes
-  @SuppressWarnings("squid:S00116")
   private static class SecurityStandards {
     int[] CWE = {};
-    String[] OWASP = {};
+
+    @SerializedName("OWASP Top 10 2021")
+    String[] OWASP_2021 = {};
```

Comment:
Can this be `Owasp_2021` to fix the code smell? With `SerializedName`, the actual name should not matter.